apps/server/src/shared/infra/identity-management/keycloak/uc/Keycloak-management.uc.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import KeycloakAdminClient from '@keycloak/keycloak-admin-client';
+import UserRepresentation from '@keycloak/keycloak-admin-client/lib/defs/userRepresentation';
+import { v1 } from 'uuid';
+import { IKeycloakSettings, KeycloakSettings } from '../interface/keycloak-settings.interface';
+import { KeycloakManagementUc } from './Keycloak-management.uc';
+import { KeycloakAdministrationService } from '../keycloak-administration.service';
+
+describe('KeycloakManagementUc', () => {
+	let module: TestingModule;
+	let uc: KeycloakManagementUc;
+	let client: KeycloakAdminClient;
+	let settings: IKeycloakSettings;
+
+	// TODO check logic
```

Comment:
TODO for this ticket or future us?

Reply:
Definitely this ticket. Thanks for pin pointing me here 😄